enchant.lic

```diff
@@ -0,0 +1,181 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#enchant
+=end
+
+custom_require.call(%w[common common-arcana events common-crafting events])
+
+class Enchant
+  include DRC
+  include DRCC
+  include DRCA
+
+  def initialize
+    @settings = get_settings
+    @book_type = "artificing"
+    @bag = @settings.crafting_container
+    @bag_items = @settings.crafting_items_in_container
+    @belt = @settings.enchanting_belt
+    arg_definitions = [
+      [
+        { name: 'chapter', regex: /\d+/i, variable: true, description: 'Chapter containing the item.' },
+        { name: 'recipe', display: 'recipe name', regex: /^[A-z\s\-']+$/i, variable: true, description: 'Name of the recipe, wrap in double quotes if this is multiple words.' },
+        { name: 'noun', regex: /\w+/i, variable: true }
+      ]
+    ]
+    #/'  Fix my syntax highlighting
```

Comment:
❔

Reply:
Notepad++ highlighting bug.  I needed that line to get syntax highlighting working below line 25.  Meant to remove it